server/apps/users/helpers.py

```diff
@@ -71,6 +74,17 @@ def delete_user(user, delete_oh_data):
     if delete_oh_data:
         ohmember.delete_all_files()
 
+    # de-auth OH member if not doing so by default
+    deauth_on_delete = getattr(settings, 'OPENHUMANS_DEAUTH_ON_DELETE', True)
+    if deauth_on_delete == False:
+        deauth_oh_member(ohmember)
+
     # Delete the actual user
     user.delete()
 
+def deauth_oh_member(oh_member):
+    deauth_url = 'https://www.openhumans.org/api/direct-sharing/project/remove-members/'
```

Comment:
Following the example in `server/apps/main/tests/jh_tests.py` it would seem to make sense to use `OPENHUMANS_OH_BASE_URL` here too.
```
from openhumans.settings import openhumans_settings
OH_BASE_URL = openhumans_settings['OPENHUMANS_OH_BASE_URL']
...
    deauth_url = urljoin(OH_BASE_URL,
                         '/api/direct-sharing/project/remove-members/')
...
```

Reply:
Great point, I had totally forgotten that we have that setting!